web/config/builder.go

```diff
@@ -57,10 +70,15 @@ func AddFlags(flags *pflag.FlagSet) {
 	flags.StringP(scopesFilePath, "s", "", "Sets the path to the json file where the available scopes will be found")
 	flags.StringP(databaseURL, "d", "", "Sets the database url where user credential data will be stored")
 	flags.StringP(secretKey, "k", "", "Sets the secret key used to hash the stored passwords")
+	flags.StringP(mailUser, "", "", "Sets the mail worker user")
+	flags.StringP(mailPassword, "", "", "Sets the mail worker user's password")
+	flags.StringP(mailIdentity, "", "", "Sets the mail worker user's identity")
```

Comment:
@claudiosegala, please provide an example of a user's identity. It can be confusing with the `mailUser` input